src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+@import './variables';
+
+.card {
+  width: 200px;
+  flex-shrink: 0;
+  background-color: $white;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid $gray-elements;
+  display: flex;
+  flex-direction: column;
+
+  transition: all 300ms ease-in-out;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__content {
+    margin: 40px 16px 16px;
+    align-items: center;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &:hover .card__title {
+    color: $title-hover-color;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    margin-top: 4px;
```

Comment:
👍 

You started using margin-top to position elements and continued doing so. 
You can't use gap here since there is different spacing between elements, so using margin is a valid approach. 

Margin-top property is used consistently and never mixed with margin-bottom. Good job.